test-automation-framework/org.wso2.carbon.automation.extensions/src/main/java/org/wso2/carbon/automation/extensions/FrameworkExtensionUtils.java

```diff
@@ -39,6 +39,10 @@ public static String getSystemResourceLocation() {
         return resourceLocation;
     }
 
+    public static String appendToSystemResourceLocation(String pathToAppend) {
+        return getSystemResourceLocation() + pathToAppend;
```

Comment:
This method is a bit error-prone. What happens if pathToAppend did not start with a file separator? 

Shall we canonicalize the path?

Reply:
Please re-think the method name as well.